src/igvfd/tests/fixtures/schemas/measurement_set.py

```diff
@@ -275,3 +275,17 @@ def measurement_set_with_protocols(testapp, lab, award, assay_term_starr, tissue
         'preferred_assay_title': 'SUPERSTARR'
     }
     return testapp.post_json('/measurement_set', item).json['@graph'][0]
+
+
+@pytest.fixture
+def measurement_set_with_functional_assay_mechanisms(testapp, lab, award, assay_term_starr, tissue, phenotype_term_from_go):
+    item = {
+        'award': award['@id'],
+        'lab': lab['@id'],
+        'assay_term': assay_term_starr['@id'],
+        'samples': [tissue['@id']],
+        'file_set_type': 'experimental data',
+        'preferred_assay_title': 'SUPERSTARR',
+        'functional_assay_mechanisms': [phenotype_term_from_go['@id']]
+    }
+    return testapp.post_json('/measurement_set', item).json['@graph'][0]
```

Comment:
Do we need to add status=201 here? Interestingly none of the measurement-set fixtures have this.